packages/esm-patient-conditions-app/src/conditions/conditions-form.component.tsx

```diff
@@ -1,104 +1,116 @@
+    mode: 'all',
+    resolver: zodResolver(conditionSchema),
+    defaultValues: {
+      onsetDateTime:
+        formContext == 'editing'
+          ? matchingCondition?.onsetDateTime
+            ? new Date(matchingCondition?.onsetDateTime)
+            : null
+          : null,
+      clinicalStatus: condition?.cells?.find((cell) => cell?.info?.header === 'clinicalStatus')?.value ?? 'Active',
+      search: condition?.cells?.find((cell) => cell?.info?.header === 'display')?.value,
+    },
+  });
 
-      submissionNotifier.next({ isSubmitting: true });
+  const onSubmit = (data) => {
+    setIsSubmittingForm(true);
+  };
 
-      if (!!errorCreating || !!errorUpdating) {
-        submissionNotifier.next({ isSubmitting: false });
-      }
-    },
-    [errorCreating, errorUpdating, submissionNotifier],
-  );
+  const onError = (error) => {
+    setIsSubmittingForm(false);
+    console.error(error);
+  };
 
   return (
-    <Form className={styles.form} onSubmit={handleSubmit}>
-      <ConditionsWidget
-        patientUuid={patientUuid}
-        closeWorkspace={closeWorkspace}
-        conditionToEdit={condition}
-        formContext={formContext}
-        setErrorCreating={setErrorCreating}
-        setErrorUpdating={setErrorUpdating}
-        setHasSubmissibleValue={setHasSubmissibleValue}
-        submissionNotifier={submissionNotifier}
-      />
-      <div>
-        {errorCreating ? (
-          <div className={styles.errorContainer}>
-            <InlineNotification
-              className={styles.error}
-              role="alert"
-              kind="error"
-              lowContrast
-              title={t('errorCreatingCondition', 'Error creating condition')}
-              subtitle={errorCreating?.message}
-            />
-          </div>
-        ) : null}
-        {errorUpdating ? (
-          <div className={styles.errorContainer}>
-            <InlineNotification
-              className={styles.error}
-              role="alert"
-              kind="error"
-              lowContrast
-              title={t('errorUpdatingCondition', 'Error updating condition')}
-              subtitle={errorUpdating?.message}
-            />
-          </div>
-        ) : null}
-        <ButtonSet className={isTablet ? styles.tablet : styles.desktop}>
-          <Button className={styles.button} kind="secondary" onClick={() => closeWorkspace()}>
-            {t('cancel', 'Cancel')}
-          </Button>
-          <Button
-            className={styles.button}
-            disabled={!hasSubmissibleValue || (hasSubmissibleValue && isSubmittingForm)}
-            kind="primary"
-            type="submit"
-          >
-            {isSubmittingForm ? (
-              <InlineLoading description={t('saving', 'Saving') + '...'} />
-            ) : (
-              <span>{t('saveAndClose', 'Save & close')}</span>
-            )}
-          </Button>
-        </ButtonSet>
-      </div>
-    </Form>
+    <FormProvider {...methods}>
```

Comment:
Passing down form contexts and  methods  
[ api](https://react-hook-form.com/docs/formprovider)